.golangci.yml

```diff
@@ -12,6 +12,13 @@ issues:
   # Default: 3
   max-same-issues: 0
   new-from-rev: ""
+  exclude-rules:
+    - path: '(.+)_test\.go'
+      linters:
+        - funlen
+        - goconst
+        - govet
```

Comment:
I understand the other disabled linters, but `govet` can catch useful things in tests.
> Printf calls whose arguments do not align with the format string

I'm sure we get warnings we don't care about, but we will miss bugs that affect test correctness. For example the `loopclosure` functionality in `govet` would catch the data race mentioned in my other comments.

Reply:
Agreed. Thanks